docs/0.1/index.md

```diff
@@ -449,45 +434,48 @@ Reference: [prev](https://www.w3.org/TR/activitystreams-vocabulary/#dfn-prev) pr
 
 A link to the previous _Change Set_{:.term} in this _Entry Point_{:.term} for the _Entity Collection_{:.term}.
 
-The _Change Set_{:.term} _MAY_{:.strong-term} have a _prev_{:.term} property if there are
+The _Change Set_{:.term} c property if there are
```

Comment:
Should this be:

The _Change Set_{:.term} _MAY_{:.strong-term} have a `prev` property if there are

Reply:
Yes, missed that one, will fix